### PR TITLE
Collections implementation

### PR DESCRIPTION
**Pull request by deepakghadge19**
added code for HashSet, TreeSet, HashMap, TreeMap

### CODE REVIEW

File: src/collections_16March2023/HashSetFeature.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package collections_16March2023;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class HashSetFeature {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Set<String>hashset = new HashSet<>();
```
Comment (amitsahu8) [inline src/collections_16March2023/HashSetFeature.java:10]: remove the String datatype from set and try to print

File: src/collections_16March2023/LinkedListFeature.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package collections_16March2023;
+
+import java.util.LinkedList;
+
+public class LinkedListFeature {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+			LinkedList<String> list = new  LinkedList<>();
+			list.add("sun");
```
Comment (amitsahu8) [inline src/collections_16March2023/LinkedListFeature.java:10]: code format issue

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/collections_16March2023/HashSetFeature.java (hunk 1 above)
Comment (deepakghadge19) [reply src/collections_16March2023/HashSetFeature.java]: Done

File: src/collections_16March2023/HashMapFeature.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package collections_16March2023;
+
```
Comment (amitsahu8) [inline src/collections_16March2023/HashMapFeature.java:2]: remove the .class files

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

**Review by amitsahu8 — APPROVED** [review]

**Review by rohitlamsonge66 — APPROVED** [review]

**Pull request merged**